ordersettler/ordersettler.go

```diff
@@ -326,7 +327,7 @@ func (r *OrderSettler) SettleBatch(ctx context.Context, batch types.SettlementBa
 			}
 
 			if rawTx == "" {
-				continue
+				return nil
```

Comment:
lets actually just remove this rawTx check and add an additional check right after the InitiateBatchSettlement call that throws an error if rawTx is the empty string. we should not be returning prematurely from this batch loop as we need to ensure that the state for all of the settlements in the batch is updated.